charts/chainlink-cluster/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: chainlink-cluster
 description: Chainlink nodes cluster
-version: 0.4.0
+version: 0.4.2
```

Comment:
would it be `0.4.1`?

Reply:
Normally, yes. Keeping it in sync with the version we use on CCIP though. Eventually this will the canonical one.